SPECS/libsepol/libsepol.spec

```diff
@@ -66,15 +66,6 @@ rm -f %{buildroot}%{_bindir}/chkcon
 rm -rf %{buildroot}%{_mandir}/man8
 rm -rf %{buildroot}%{_mandir}/ru/man8
 
-%check
```

Comment:
Why were the tests removed?

Reply:
The logic to unpack the "checkpolicy" project and run the tests does not make sense for libsepol. Instead, I just added the proper check section to our checkpolicy package which actually has the sources by default. 

See checkpolicy.spec change: https://github.com/microsoft/azurelinux/pull/7733/files#diff-329a0da3241ec2721fe29b0f1c416d30bb4d6191c292a1a514ccd5368ff6ac69R50

Reply:
So the same checks are being run, just in checkpolicy.